src/apps/options/main.js

```diff
@@ -4,12 +4,16 @@
 
 let React = require('react');
 let renderComponent = require('../../common/render');
+let loadTypekit = require('../../common/typekit');
 
 /**
  * Dependencies.
  */
 
-let Options = require('./components/options.react');
+let Options = require('./components/Options.react');
+
+// load Typekit
+loadTypekit();
```

Comment:
FYI we got now an license for self hosted fonts - you can fetch it from the webapp `/app/assets/fonts`